chainer/function_hooks/timer.py

```diff
@@ -132,11 +133,38 @@ def _humanized_time(self, second):
             second *= 1000.0
         return '%.2f%s' % (second, 'ns')
 
-    def print_report(self, file=sys.stdout):
-        """Prints a summary report of time profiling in functions."""
+    def _align(self, second):
+        """Returns align info: (factor, unit)."""
+        factor = 1
+        for unit in ['sec', 'ms', 'us']:
+            if second * factor >= 1:
+                return factor, unit
+            factor *= 1000.0
+        return factor, 'ns'
```

Comment:
The logic looks duplicated with `_humanized_time`. Could you refactor them?

Reply:
Integrated them into `_choose_unit`.